releases-beta/how-to-guides/create-custom-environment-variables.md

```diff
@@ -0,0 +1,56 @@
+# Create Custom Environment Variables
```

Comment:
For external use, we should call it something like "Workflow Parameters" instead. "Environment variables" only applies to the case of Buildkite and we internally used this term, but it's actually not accurate.